main.py

```diff
@@ -190,6 +191,7 @@ def main(args):
             sampler_train = torch.utils.data.DistributedSampler(
                 dataset_train, num_replicas=num_tasks, rank=global_rank, shuffle=True
             )
+        sampler_val = OrderedDistributedSampler(dataset_val, num_replicas=num_tasks, rank=global_rank)
```

Comment:
Even though now we always have `if True` in the branch, ideally you should also have the `sampler_val` be defined in the other branch (as a `SequentialSampler`)

Reply:
Thank you for the review! Have changed them.